.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
```

Comment:
Is this intentional ?

Reply:
I've added it to .gitignore because the repo didn't have any
and .DS_Store files are created on macOS in every folder that you open with Finder